packages/web3-providers/src/types/Trending.ts

```diff
@@ -116,6 +129,39 @@ export namespace TrendingAPI {
         volume_all?: number
     }
 
+    export interface Collection {
```

Comment:
Place it `packages/web3-providers/src/types/NonFungibleToken.ts`.

Reply:
done